.github/workflows/python.yml

```diff
@@ -20,15 +20,18 @@ jobs:
         with:
           python-version: ${{ env.PYTHON_VERSION }}
 
+      - name: Install pipenv
+        run: pip install pipenv
+
       - name: install dependencies
-        run: pip install -r requirements.txt
+        run: pipenv install --dev
 
       - name: run black formatter
-        run: black --check .
+        run: pipenv run black --check .
 
       - name: run flake8 linter
         # Ignore linting errors that the black formatter is opiniated about
-        run: flake8 --ignore E501,W503 .
+        run: pipenv run flake8 --ignore E501,W503 .
 
       - name: run bandit security scan
```

Comment:
@lcartey is there a specific reason that we run a bandit scan on top of the CodeQL scans that we have setup in the repo? If there isn't, I would suggest we remove bandit step in this workflow and therefore the obsolete `bandit` package from the new `Pipfile`

Reply:
Agreed, I will remove it.